addon/templates/global-landing-page.hbs

```diff
@@ -0,0 +1,70 @@
+<div class="uk-padding">
+  <h1 class="uk-text-center">
+    {{t "ember-gwr.globalLandingPage.heading"}}
+  </h1>
+  <h4 class="uk-text-center uk-margin-remove-top uk-margin-large">
+    {{t "ember-gwr.globalLandingPage.infoText"}}
+  </h4>
+
+  <div uk-grid class="uk-child-width-1-1 uk-child-width-1-3@s">
```

Comment:
Test with small width

Reply:
Works, if the screen is reeeeally small they just get stacked atop one another.